### PR TITLE
Add missing fields for PiPy to pyproject.toml

### LINKED ISSUE #1: README does not appear on Pypi
https://pypi.org/project/ferm-docker/#description

### PR DESCRIPTION
**Pull request by fleaz**
According to e.g. [this blogpost](https://johnfraney.ca/posts/2019/05/28/create-publish-python-package-poetry/) there where some fields missing to tell PiPy where to find the repo and the readme.  This should fix #1

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,9 +1,16 @@
 [tool.poetry]
 name = "ferm-docker"
 version = "0.1.0"
-description = "Automatically generate ferm firewall configurations for a local docker instance"
+description = "Automatically generate ferm firewall configuration for a local Docker instance"
 authors = ["Martin Weinelt"]
 license = "GPL-2.0-only"
+readme = "README.rst"
+homepage = "https://github.com/mweinelt/ferm-docker",
+repository = "https://github.com/mweinelt/ferm-docker",
```
Comment (mweinelt) [inline pyproject.toml]: Syntax highlighting doesn't look too  happy about the commata.

**Review by mweinelt — CHANGES_REQUESTED** [review]
Thanks, I really wonder why poetry doesn't do the README part, since it actually created README.rst for me.

**Comment by mweinelt** [comment]
Please add `Fixes #1` to the commit message.

File: pyproject.toml (hunk 1 above)
Comment (fleaz) [reply pyproject.toml]: Oh indeed, these shouldn't be there :see_no_evil:

**Comment by fleaz** [comment]
Fixed

**Review by mweinelt — APPROVED** [review]

**Comment by mweinelt** [comment]
Thank you!

**Pull request merged**